editor/plugins/editor_preview_plugins.cpp

```diff
@@ -470,10 +470,10 @@ Ref<Texture> EditorScriptPreviewPlugin::generate(const RES &p_from) {
 	if (EditorSettings::get_singleton()->get("text_editor/theme/color_theme") == "Adaptive") {
 		Ref<Theme> tm = EditorNode::get_singleton()->get_theme_base()->get_theme();
 
-		bg_color = tm->get_color("text_editor/highlighting/background_color", "Editor");
-		keyword_color = tm->get_color("text_editor/highlighting/keyword_color", "Editor");
-		text_color = tm->get_color("text_editor/highlighting/text_color", "Editor");
-		symbol_color = tm->get_color("text_editor/highlighting/symbol_color", "Editor");
+		bg_color = tm->get_color("background_color", "Editor");
```

Comment:
the reason why this fails is because those values are not in the theme, the proper solution to this would be to get the values from the ProjectSettings (where they are actually defined), doing
```
bg_color = EditorSettings::get_singleton()->get("text_editor/highlighting/background_color");
keyword_color = EditorSettings::get_singleton()->get("text_editor/highlighting/keyword_color");
text_color = EditorSettings::get_singleton()->get("text_editor/highlighting/text_color");
symbol_color = EditorSettings::get_singleton()->get("text_editor/highlighting/symbol_color");
```
instead, and remove the redundant definitions in editor_themes.cpp